packages/terra-consumer-nav/src/Nav.scss

```diff
@@ -1,7 +1,67 @@
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,400i,700');
+@import './variables';
+@import './text_utility';
+
 :local {
-  /* stylelint-disable selector-class-pattern  */
+  @keyframes fadeIn {
+    from {left: -$terra-consumer--nav-container-width;}
+    to {left: 0;}
+  }
+
+  @keyframes fadeOut {
+    from {left: 0;}
+    to {left: -$terra-consumer--nav-container-width;}
+  }
+
+  body {
+    font-family: 'Roboto', sans-serif;
+    margin: 0;
+  }
+
   .nav {
+    height: 100%;
```

Comment:
played around with this, and height technically isn't even a necessary property since min-height is set to full height of the viewport